### ISSUE TITLE
[auth] cannot authenticate after setting passphrase

### ISSUE THREAD

**Issue by Saviq** [canonical/multipass#2469]
```plain
$ multipass version
multipass   1.8.0-rc.428+g034aca28
multipassd  1.8.0-rc.428+g034aca28
$ sudo multipass set local.passphrase=foo
$ multipass auth foo
authenticate failed: Passphrase is not correct. Please try again.
```

**Comment by Saviq** [comment]
Problem introduced here:

https://github.com/canonical/multipass/commit/fc7d0f7f#diff-b3bc64a5be31e4d50cbe8e94a4913bcb0237fb94285fb10dde0383c5c6805b56R127-R130

**Comment by townsend2010** [comment]
Fixed by #2471
